### PR TITLE
IWF-141: Disable global version search attribute

### LINKED ISSUE #427: Disable global version search attribute(IwfGlobalWorkflowVersion) to save an action
Golang SDK has already a built in search attribute “TemporalChangeVersion” to use, and it’s free in the cloud. (note: this is not available in Java SDK https://github.com/temporalio/sdk-java/issues/587#issuecomment-2398907995 ) . Therefore, it is okay to disable our search attribute. 

This needs versioning if doing by default

### PR DESCRIPTION
**Pull request by lwolczynski**
### Checklist
- [ ] Code compiles correctly
- [ ] Tests for the changes have been added
- [ ] All tests passing
- [ ] **This PR change is backwards-compatible**
- [ ] **This PR CONTAINS a (planned) breaking change** (it is not backward compatible)

### Related Issue
Closes #427

### CODE REVIEW

File: service/interpreter/globalVersioner.go [hunk 1]
```diff
@@ -58,6 +59,10 @@ func (p *GlobalVersioner) IsAfterVersionOfTemporal26SDK() bool {
 	return p.version >= StartingVersionTemporal26SDK
 }
 
+func (p *GlobalVersioner) IsAfterVersionOfNoIwfGlobalVersionSearchAttribute() bool {
```
Comment (longquanzheng) [inline service/interpreter/globalVersioner.go:62]: actually, we don't have to strictly use this naming pattern. I think we could just rename the existing one to ```go
IsUsingGlobalVersionSearchAttribute(){
   return version >= .. && <= ...; 
}

File: service/interpreter/globalVersioner.go [hunk 2]
```diff
@@ -58,6 +61,16 @@ func (p *GlobalVersioner) IsAfterVersionOfTemporal26SDK() bool {
...
```
Comment (longquanzheng) [inline service/interpreter/globalVersioner.go:64]: Nit: it is not being used so can be removed. Do you put it here intentionally for consistency?

**Review by longquanzheng — APPROVED** [review]
Lgtm with a nit comment

File: service/interpreter/globalVersioner.go (hunk 2 above)
Comment (lwolczynski) [reply service/interpreter/globalVersioner.go:64]: Yes, I added it for consistency

Comment (longquanzheng) [reply service/interpreter/globalVersioner.go:64]: Yeah Sounds good. I am fine to keep it

**Pull request merged**